lib/operations/numeric/lower.js

```diff
@@ -0,0 +1,6 @@
+function lower(a, b, c) {
+  return c === undefined ? a < b : a < b && b < c;
+}
+
+lower.code = '<';
```

Comment:
Same here `code` as described in https://github.com/axa-ch/json-logic-js/pull/1/files#r259730014

`<` is an existing operator